packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/Choices.js

```diff
@@ -0,0 +1,79 @@
+import {
+  Grid
+} from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+import sortIntoColumns from 'sort-into-columns'
+import styled, { css } from 'styled-components'
+
+import ChoiceButton from './ChoiceButton'
+
+const StyledGrid = styled(Grid)`
+  ${props => props.theme.dark ?
+    css`background-color: ${props.theme.global.colors['dark-1']};` :
+    css`background-color: ${props.theme.global.colors['light-1']};`
+  }
+`
+
+function howManyColumns ({ length }) {
+  if (length <= 5) {
+    return 1
+  } else if (length <= 20) {
+    return 2
+  } else {
+    return 3
+  }
+}
+
+function Choices (props) {
+  const {
+    filteredChoices,
+    onChoose,
+    task
+  } = props
+
+  const columnsCount = howManyColumns(filteredChoices)
+  const sortedFilteredChoices = sortIntoColumns(filteredChoices, columnsCount)
+
+  return (
+    <StyledGrid
+      columns={{
+        count: columnsCount,
+        size: 'auto'
+      }}
+      gap='2px'
+    >
+      {sortedFilteredChoices.map((choiceId) => {
+        const choice = task.choices[choiceId]
+        return (
+          <ChoiceButton
+            key={choiceId}
+            choiceId={choiceId}
+            choiceLabel={choice.label}
+            onChoose={onChoose}
+          />
+        )
+      })}
+    </StyledGrid>
```

Comment:
Marking the list of choices up as a HTML list might make it easier to navigate with a screen reader.

Reply:
Great suggestion, thank you @eatyourgreens! I hope it's ok I add this to #1991? I'm hopeful it'll be easier to test accessibility, focus, keys, etc. once all the survey task pieces are wired together, and it'll keep the initial PRs, like this one, just a little simpler.

Reply:
Sounds good to me. By the way, we have a volunteer who classifies on survey projects, with audio subjects, using NVDA.

Reply:
That's awesome! Do you know which project? I'll make a note to test on.

Reply:
https://www.zooniverse.org/talk/18/1002723?comment=1884567

Reply:
There was also a volunteer with a severe mobility impairment, who emailed us via `contact@`, who dictates his survey task classifications using Dragon. Continuing to support voice control would be good.

Reply:
From memory, changes that would help in a screen reader are meaningful alt text for example images in the `Choice` component, or the ability for audio projects to embed audio files, instead of images, for each choice.

Reply:
For a sighted user, we could maybe distinguish between radio buttons and checkboxes for the answers to Choice questions. At the moment they look exactly the same, which makes them hard to navigate with the keyboard.